rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -154,7 +170,7 @@ data class StringValue(var value: String, val varying: Boolean = false) : Value
 
     fun length(varying: Boolean = this.varying): Int {
         if (varying) {
-            if (value.isBlank())
+            if (value.isEmpty())
```

Comment:
What issue does it fix?

Reply:
A string can be zero-length only if it is declared as VARYING. Any non-varying string (initialized by 'CLEAR', 'EVAL', ecc... or not initialized too) contains blank characters for its length. Instead, in case of varying string with some blank spaces (not empty at all) must not return zero-length, cause zero-length must be returned only if the varying string does not contain anything, so check 'isEmpty' is correct instead of 'isBlank'. This fix a test who scan for a blank char  and replace the blank char with %20 char (MUTE15_14.rpgle)